code/modules/projectiles/gun_attachables.dm

```diff
@@ -341,6 +341,28 @@ Defined in conflicts.dm of the #defines folder.
 			F.forceMove(loc)
 		qdel(src) //Delete da old bayonet
 
+/obj/item/attachable/bayonetknife
+	name = "M-22 bayonet"
+	desc = "A sharp knife that is the standard issue combat knife of the TerraGov Marine Corps can be attached to a variety of weapons at will or used as a standard knife."
+	icon_state = "bayonetknife"
+	item_state = "combat_knife"
+	attach_icon = "bayonetknife_a"
+	force = 25
+	throwforce = 20
+	throw_speed = 3
+	throw_range = 6
+	attack_speed = 7
+	attach_delay = 10 //Bayonets attach/detach quickly.
+	detach_delay = 10
+	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	melee_mod = 25
```

Comment:
```suggestion
	melee_mod = 20
```
Suggestion.

Reply:
Eh. I personally like it being basically a bayonet that can be put in your pocket or use as a knife. . As I replaced bayonets inside the combat vendors with this.